frontend/src/app/pages/landing-page/landing-page/landing-page.component.html

```diff
@@ -1,4 +1,31 @@
 <div class="container">
+    <!--https://www.w3schools.com/howto/howto_js_alert.asp-->
+    <div class="alert">
+        <span
+            class="closebtn"
+            onclick="this.parentElement.style.display='none';"
+            >&times;</span
+        >
+        <div class="display-6">Hinweise/Informationen</div>
+        <br />
+        <p>
+            Bitte beachten Sie, dass diese Webseite sich noch in Entwicklung
+            befindet. Es ist daher jederzeit möglich, dass sich Funktionen
+            grundlegend ändern, oder die Webseite zeitweise nicht erreichbar
+            ist. Insbesondere ist zu erwarten, dass Daten nicht jederzeit
+            verloren gehen können.
+        </p>
+        <p>
+            Weitere Informationen zu dieser Seite finden sie auf der
+            <a
+                class="text-white"
+                target="_blank"
+                rel="noopener noreferrer"
+                href="https://hpi.de/teaching/bachelor-projects/digitales-fuehrungssimulationstraining.html"
+                >Projektwebseite</a
+            >.
+        </p>
+    </div>
```

Comment:
Use Bootstrap https://getbootstrap.com/docs/5.1/components/alerts/ and angular DOM manipulation.

In this case you could also use https://ng-bootstrap.github.io/#/components/alert/examples

- You then also have to import the `NgbAlertModule` into the `landing-page-module`